api/src/main/java/io/grpc/ServerCall.java

```diff
@@ -209,6 +209,19 @@ public void setCompression(String compressor) {
     // noop
   }
 
+  /**
+   * A hint to the call that specifies how many bytes must be queued before
+   * {@link #isReady()} will return true. A call may ignore this property if
+   * unsupported. This must be set before any messages are sent.
```

Comment:
Would it be clear to change "must" to "may only". Otherwise it could be interpreted as saying this method must be called.